pkg/sloop/storemanager/storemanager.go

```diff
@@ -76,21 +80,46 @@ func (sm *StoreManager) Start() {
 				glog.Infof("Store manager main loop exiting")
 				return
 			}
-			temporaryEmitMetrics(sm.storeRoot, sm.tables.Db(), sm.fs)
+			temporaryEmitMetrics(sm.config.StoreRoot, sm.tables.Db(), sm.fs)
 			metricGcRunCount.Inc()
-			cleanupPerformed, err := doCleanup(sm.tables, sm.storeRoot, sm.timeLimit, sm.sizeLimitMb*1024*1024, sm.fs)
+			cleanupPerformed, err := doCleanup(sm.tables, sm.config.StoreRoot, sm.config.TimeLimit, sm.config.SizeLimitMb*1024*1024, sm.fs)
 			if err != nil {
-				glog.Errorf("GC failed with err:%v, will sleep: %v and retry later ...", err, sm.freq)
-				sm.sleeper.Sleep(sm.freq)
+				glog.Errorf("GC failed with err:%v, will sleep: %v and retry later ...", err, sm.config.Freq)
 			} else if !cleanupPerformed {
-				glog.V(2).Infof("GC did not need to clean anything, will sleep: %v", sm.freq)
-				sm.sleeper.Sleep(sm.freq)
+				glog.V(2).Infof("GC did not need to clean anything, will sleep: %v", sm.config.Freq)
 			} else {
 				// We did some cleanup and there were no errors.  Because we may be in a low space situation lets skip
 				// the sleep and repeat the loop
-				glog.Infof("GC cleanup performed")
+				glog.Infof("GC cleanup performed without errors, next run: %v", sm.config.Freq)
 				metricGcCleanupPerformedCount.Inc()
 			}
+			sm.sleeper.Sleep(sm.config.Freq)
```

Comment:
moving the sleep here invalidates some of the comment on lines 91 & 92

Reply:
I fixed it